library/core/registry/src/main/java/org/quiltmc/qsl/registry/mixin/RegistryMixin.java

```diff
@@ -62,4 +63,25 @@ private static void onFreezeBuiltins(CallbackInfo ci) {
 		DebugChunkGeneratorAccessor.setZSideLength(zLength);
 		//endregion
 	}
+
+	@Inject(method = "<clinit>", at = @At("TAIL"))
+	private static void quilt$markRegistriesForSync(CallbackInfo ci) {
+		SynchronizedRegistry.markForSync(
```

Comment:
Isn't it kind of unexpected behavior for new registries to not be synced by default?
Maybe a better solution is to sync all by default, and use RegistrySynchronization as an opt-out instead?

Reply:
Vanilla ones are also opt in, so it made sense to me to apply that globally, but I can change that